src/request/components/RequestDetailPanelData.tsx

```diff
@@ -0,0 +1,48 @@
+import { MasterDetailTriPanel } from './MasterDetailTriPanel';
+import { RequestDetailPanelDataOperationCommandContainer } from '../containers/RequestDetailPanelDataOperationCommandContainer';
+import { RequestDetailPanelDataOperationTableContainer } from '../containers/RequestDetailPanelDataOperationTableContainer';
+
+import * as React from 'react';
+import Highlight = require('react-highlight');
+
+export interface IDataComponentProps {
+    totalOperationCount: number;
+}
+
+export class DataComponent extends React.Component<IDataComponentProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data'>
+            {
+                this.props.totalOperationCount > 0
+                    ? <MasterDetailTriPanel masterPanel={this.renderMaster()} leftDetailPanel={this.renderLeftDetail()} leftDetailPanelTitle='Command' rightDetailPanel={this.renderRightDetail()} rightDetailPanelTitle='Summary' />
+                    : <div className='tab-section text-minor'>Could not find any data.</div>  
+            }
+            </div>
+        );
+    }
```

Comment:
Yes, I thought the same thing.  In previous teams we had a common "pass through" layout panel that would handle the getting-data/no-data/failed-to-get-data cases in a consistent way.  Definitely something to refactor towards as a separate issue.